src/stare/models/auth.py

```
@@ -0,0 +1,104 @@
+
+from pydantic import BaseModel, ConfigDict, Field
+
+from stare.models.common import _Base
+
+
+class _OAuthTokenResponse(BaseModel):
+    """Raw OAuth2 token endpoint response from CERN Keycloak."""
+
+    access_token: str
+    refresh_token: str | None = None
+    token_type: str = "Bearer"
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to: "token_type"

(S105)